tests/nodeos_contrl_c_test.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+
+from testUtils import Utils
+from datetime import datetime
+from datetime import timedelta
+import time
+from Cluster import Cluster
+import json
+from WalletMgr import WalletMgr
+from Node import Node
+from TestHelper import TestHelper
+
+import signal
+
+###############################################################
+# nodeos_contrl_c_lr_test
+#
+# This test sets up one producing nodes and one "bridge" node using test_control_api_plugin. We send
+# transactions to the bridge node. After the accounts are initialized and allocated with symbols, we
+# kill the producing node. Then we flood hundreds transactions to the bridge node. At the end, we
+# kill the bridge. It tests that no crashes happen when the nodes are killed.
+#
+###############################################################
+
+errorExit=Utils.errorExit
+
+args = TestHelper.parse_args({"--wallet-port", "-v"})
+
+cluster=Cluster(walletd=True)
+killAll=True
+totalProducerNodes=1
+totalNonProducerNodes=1
+totalNodes=totalProducerNodes+totalNonProducerNodes
+walletPort=args.wallet_port
+walletMgr=WalletMgr(True, port=walletPort)
+producerEndpoint = '127.0.0.1:8888'
+httpServerAddress = '127.0.0.1:8889'
+
+try:
+    TestHelper.printSystemInfo("BEGIN")
+    cluster.setWalletMgr(walletMgr)
+    cluster.killall(allInstances=killAll)
+    cluster.cleanup()
+
+    specificExtraNodeosArgs = {}
+    specificExtraNodeosArgs[0] = "--plugin eosio::producer_plugin --plugin eosio::chain_api_plugin --plugin eosio::http_plugin "
+    "--plugin eosio::txn_test_gen_plugin --plugin eosio::producer_api_plugin --plugin eosio::history_plugin "
+    "--plugin eosio::history_api_plugin --plugin eosio::state_history_plugin --plugin eosio::db_size_api_plugin "
+    "--p2p-listen-endpoint=0.0.0.0:9876 --p2p-server-address=0.0.0.0:9876 --http-server-address=127.0.0.1:8888 "
+    "--delete-all-blocks --disable-replay-opts --delete-state-history --access-control-allow-origin='*'"
```

Comment:
You don't need history_plugin, history_api_plugin, state_history_plugin, db_size_api_plugin.

I'm not sure you even need anything here for extra args. @brianjohnson5972

Reply:
> You don't need history_plugin, history_api_plugin, state_history_plugin, db_size_api_plugin.
> 
> I'm not sure you even need anything here for extra args. @brianjohnson5972

Removed the extra options